src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -144,6 +145,17 @@ def _extract_messages(self) -> None:
                     exc_info=True,
                 )
                 continue
+            if (
+                datetime.fromtimestamp(parsed_payload["timestamp"], tz=timezone.utc)
+                - datetime.now(tz=timezone.utc)
+            ) > timedelta(days=5):
+                self.skipped_offsets.add(partition_offset)
+                logger.error(
+                    "process_messages.invalid_timestamp",
+                    extra={"payload_value": str(msg.payload.value)},
+                    exc_info=True,
+                )
```

Comment:
Two issues:
- this should not be an error. Warning at best. A metric would even be bette, you do not need all the additional context of a sentry event.
- Please do not log every dropped event. That is basically a DDOS vector as a large volume of events that generate sentry events add enough load to the system that it will impact the throughput. See INC-356

Reply:
Fixed, thanks!